src/contexts/Pools/BondedPools/index.tsx

```diff
@@ -254,16 +256,16 @@ export const BondedPoolsProvider = ({
    * Determine bonded pool's current nomination statuse
    */
   const getPoolNominationStatusCode = (statuses: NominationStatuses | null) => {
-    let status = 'waiting';
+    let status = t('contexts.waiting');
```

Comment:
I don't think `contexts` is the correct key to use here.

Reply:
These are keys and do not need to be translated.

Reply:
please check the updates